src/pages/Search/Search.jsx

```diff
@@ -0,0 +1,142 @@
+import "./Search.css";
+import { useState } from "react";
+import { useCart } from "../../context/CartContext";
+import { useSearch } from "../../context/SearchContext";
+import { useWishList } from "../../context/WishlistContext";
```

Comment:
Suggestion: you could add all the context in one index.js file and import here in one line